quickstarts/README.md

```diff
@@ -0,0 +1,9 @@
+# Anthos Service Mesh Quickstarts
+
+This folder contains the quickstart material for creating a GKE cluster with Google's Managed Anthos Service Mesh.
+The following are methods showcased:
+* Terraform
+
+To view the other methods for setting up Managed Anthos Service Mesh:
+* [Fleet API](https://cloud.google.com/service-mesh/docs/managed/auto-control-plane-with-fleet)
+* [GCP Console](https://cloud.google.com/service-mesh/docs/install-anthos-service-mesh-console)
```

Comment:
**Thought:** I remember our rationale (from a previous sync) for not placing quickstart-related sample code inside the `tutorial-samples`: quickstarts don't necessarily rely on `cloud.google.com` tutorials. But now I'm thinking, should we make sure there is always a `cloud.google.com` for each quickstart in this repo — this is in line with the DPE philosophy of letting Tech Writers own the instructions/words. In this case, we could stuff `terraform-asm-mcp` into `tutorial-samples`.

Reply:
same point to above, we can discuss it off line and see where we want to go with this :)